[DOCS] Reformats cluster node info API (#45446)
elastic/elasticsearch: docs/reference/cluster/nodes-info.asciidoc

		process
---
	
`process`::
		Process statistics, memory consumption, cpu usage, open file descriptors.

`settings`::
		

`thread_pool`::
		Statistics about each thread pool, including current size, queue and 
		rejected tasks
		
`transport`::
		Transport statistics about sent and received bytes in cluster communication.
--

include::{docdir}/rest-api/common-parms.asciidoc[tag=node-id]

[[cluster-nodes-info-api-response-body]]
==== {api-response-body-title}

`build_hash`::
		Short hash of the last git commit in this release.

`host`::
		The node's host name.

`ip`::
		The node's IP address.

`name`::
		The node's name.

`total_indexing_buffer`::
		Total heap allowed to be used to hold recently indexed
		documents before they must be written to disk.  This size is
		a shared pool across all shards on this node, and is
    controlled by <<indexing-buffer,Indexing Buffer settings>>.

`total_indexing_buffer_in_bytes`::
		Same as `total_indexing_buffer`, but expressed in bytes.

`transport_address`::
		Host and port where transport HTTP connections are accepted.

`version`::
		{es} version running on this node.

> The `os` flag can be set to retrieve information that concern the operating 

Reviewer: ```suggestion

```

> system:

Reviewer: ```suggestion
system:
```

// TESTRESPONSE[s/"plugins": \[[^\]]*\]/"plugins": $body.$_path/]
// TESTRESPONSE[s/"modules": \[[^\]]*\]/"modules": $body.$_path/]

> 

Reviewer: Not required, but you may want to consider adding a heading here to better separate the two examples.

> If `ingest` is specified, the result will contain details about the available 

Reviewer: ```suggestion
If `ingest` is specified, the response contains details about the available 
```
